upload-api/buckets/invocation-store.js

```diff
@@ -116,12 +116,12 @@ export const useInvocationStore = (s3client, bucketName) => {
       })
       const listObject = await s3client.send(listObjectCmd)
       const carEntry = listObject.Contents?.find(
-        content => content.Key?.endsWith('.workflow')
+        content => content.Key?.endsWith('.out')
```

Comment:
😱  this had a bug, but this code path was not being used. See https://github.com/web3-storage/w3infra/blob/main/docs/ucan-invocation-stream.md#buckets as this first was `.workflow` but then was iterated to `.in` and `.out`